### ISSUE TITLE
Tutorial on deploying Pangeo-like dask-gateway on Jetstream

### ISSUE THREAD

**Issue by zonca** [zonca/jupyterhub-deploy-kubernetes-jetstream#39]
@julienchastang
The Pangeo helm chart is basically joining together the dask-gateway helm chart and the JupyterHub helm-chart.

I extracted the minimum necessary from their configuration to add `dask-gateway` to my standard deployment.
The only extra thing if you want a full Pangeo deployment is to use the Pangeo images.

See https://zonca.dev/2020/08/dask-gateway-jupyterhub.html

**Comment by julienchastang** [comment]
Thank you for your efforts. I will try to scrutinize this soon. I am deploying seven JupyterHubs this week for the fall semester.

**Comment by julienchastang** [comment]
I am finally able to look at this. I am looking for a sanity check / confirmation that the

```yaml
     url: http://traefik-dask-gateway
```

value is correct and does not need to be altered/changed in the dask yaml files.

Thanks again.

**Comment by zonca** [comment]
Correct

**Comment by julienchastang** [comment]
OK, thanks. I can confirm these instructions work, but I did run into a few issues:

1. Inserting a custom docker container in [secrets.yaml](https://github.com/julienchastang/jupyter-classroom/blob/master/farallon-fall-2020/secrets.yaml) does not seem to have any effect when I access the Hub. I do not see my kernel in the Kernel menu nor do I see some of the utilities I installed in the container (e.g., `curl`).
2. The example provided is, I believe, incomplete. The `options` variable is not defined. 
3. I think a little extra verbiage should be added as to where the token should be inserted in the dask yaml files. E.g., just look for `TOKEN`.

Thanks again.

Update: Had to make many edits for grammar :-)
